src/main/java/app/Duke.java

```diff
@@ -0,0 +1,58 @@
+package app;
+
+import app.exceptions.BaseException;
+import app.core.UserInterface;
+import app.core.commands.Command;
+import app.core.commands.CommandManager;
+import app.core.tasks.TaskManager;
+
+/**
+ * Main class for Duke. The class contains all the main components 
+ * required for Duke to run properly.
+ */
+public final class Duke {
+    /**
+     * Welcome message that is printed upon starting Duke
+     */
+    public static final String WELCOME_MESSAGE = "Wussup Dawggg! I'm Dukeee\nWhat you want me do?";
```

Comment:
I like that you made your strings constants. Also, pretty dope comments adds personality to Duke.